Sources/KeyboardShortcuts/Shortcut.swift

```diff
@@ -153,6 +153,12 @@ extension KeyboardShortcuts.Shortcut {
 	}
 }
 
+extension KeyboardShortcuts.Key {
+	public var specialCharacterMapping: String? {
+		keyToCharacterMapping[self]
+	}
+}
```

Comment:
?

Reply:
Because I wanted that mapping for other uses - and since it was there, I thought why reinvent the wheel?

Reply:
I'll rename it to make it more obvious. Basically I'm using it in an app to take a key and return a nice string. Since the mapping is already in place (just private) I thought to expose a method to get it.   If however keyToCharacter() were public, I wouldn't need the additional extension at all and could just remove it.

Reply:
That's an unrelated change and should be done in a separate pull request.